mx.jruby/mx_jruby.py

```diff
@@ -81,22 +82,22 @@ def build(self):
         truffle = mx.suite('truffle')
         truffle_commit = truffle.vc.parent(truffle.dir)
 
-        mx.run_mx(['maven-install'], suite=truffle)
+        mx.run_mx(['maven-install', '--repo', mavenDir], suite=truffle)
 
         open(os.path.join(rubyDir, 'VERSION'), 'w').write('graal-vm\n')
 
         # Build jruby-truffle
 
-        mx.run(['find', '.'], nonZeroIsFatal=False, cwd=rubyDir)
-        mx.run_maven(['--version'], nonZeroIsFatal=False, cwd=rubyDir)
+        #mx.run(['find', '.'], nonZeroIsFatal=False, cwd=rubyDir)
```

Comment:
Please remove code instead of commenting it, git can always find it back should we need it.